packages/core/src/utils/node.ts

```diff
@@ -0,0 +1,12 @@
+import cheerio from 'cheerio';
+import { DomElement } from 'htmlparser2';
+
+export type TextElement = DomElement;
+
+export type MbNode = DomElement & cheerio.Element & {
```

Comment:
Would be nice to document the rationale for this in the code, as it might cause some confusion as to why such an abstraction exists. Same for the one below.